api/api.go

```diff
@@ -42,6 +62,18 @@ func New() *API {
 		lookupAddr:    net.LookupAddr,
 		lookupCountry: func(ip net.IP) (string, error) { return "", nil },
 		ipFromRequest: ipFromRequest,
+		client: &http.Client{
```

Comment:
Since the parameter is no longer an URL, maybe this should just be a regular TCP connection instead? That way it can be used to check if any kind of service.